syntax/scss.vim

```diff
@@ -143,9 +143,19 @@ syn match scssReturn "@return" contained
 syn match scssExtend "@extend" nextgroup=scssExtendedSelector skipwhite containedin=cssMediaBlock
 syn match scssExtendedSelector "[^;]\+" contained contains=cssTagName,cssPseudoClass,scssOptional,scssSelectorChar skipwhite
 syn match scssOptional "!optional" contained
-syn match scssImport "@import" nextgroup=scssImportList
+
+syn match scssImport "@import" nextgroup=scssUrlList
```

Comment:
What's the reason for this change? Did you forget to add the definition for `scssUrlList`? This is the only line it appears on.

Reply:
Sorry, that was an artefact left over from an earlier version of the implementation where there was a shared group used by @import, @forward and @use, before I realized I'd have to implement "as" and "with" for @forward and @use. It should be "scssImportList". I've just pushed a commit which fixes this.